src/dashboards/videos/details.jsonnet

```diff
@@ -449,3 +458,77 @@ dashboard.new(
   ),
   gridPos={ x: 12, y: 8, w: 12, h: 9 }
 )
+.addPanel(
+  {
+    title: 'Event distribution during the video',
+    description: |||
+      We divide the video duration into equal intervals and for each interval display
+      its event distribution.
+      The interval size is controlled by the `Event group interval` variable
+    |||,
+    aliasColors: {},
+    avgLineShow: false,
+    barValuesShow: true,
+    cacheTimeout: null,
+    chartType: 'stacked bar chart',
+    colorSch: [],
+    datasource: 'lrs',
+    fontColor: '#fff',
```

Comment:
Apropos, we can control the font color, do we set it to `#000` for the light grafana theme ?

Reply:
Nope. Leave it as is for now. You'll do better in your own plugin using this: https://grafana.com/docs/grafana/latest/packages_api/ui/withtheme/

:wink: